avm/__init__.py

```diff
@@ -298,7 +300,8 @@ def _trigger(self, call_from, call_to, time, callid, event, branch):
                 self._call_incoming_cid = callid
 
                 # reset duration for incoming calls
-                self._duration_item['call_duration_incoming'](0, self._plugin_instance.get_shortname())
+                if not self._duration_item['call_duration_incoming'] is None:
```

Comment:
Is this really working (did u check it?)? 
Normally I check for the existence of keys in dicts with "if 'call_duration_incoming' in self._duration_item: ..."

So I am not sure if the None comparison works, because I think its trying to access the non existing key before comparing?

Reply:
ok no i see, you are initializing it with None above.. then ok..

Reply:
Hmm I think I would still prefer to fix it with the
if 'call_duration_incoming' in self._duration_item
check...
In my view a non existing key signals better the item does not exist than one with "None" as value.

Reply:
I've added the checking for None because this is done often in the code.

Reply:
just checked.. you may be right.. accepting the PR